go.mod

```diff
@@ -243,6 +244,7 @@ require (
 	github.com/Azure/go-autorest/tracing v0.6.0 // indirect
 	github.com/Azure/go-ntlmssp v0.0.0-20221128193559-754e69321358 // indirect
 	github.com/AzureAD/microsoft-authentication-library-for-go v1.0.0 // indirect
+	github.com/IBM/nzgo v11.1.0+incompatible // indirect
```

Comment:
So you are adding v12 and v11? I think all v11 lines in go.mod and go.sum can be removed?

Reply:
Thanks - I'll fix that

Reply:
To be clear, this is an indirect dependency from github.com/IBM/nzgo/v12 and so to remove it from here, I needs to be removed from nzgo/v12 first. I'll open a PR for nzgo with such a change.